.github/scripts/bsd_tests.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
```

Comment:
If bash command is not found, does this work?

Reply:
If we want to use bash, we need to specify the correct path (`/usr/bin/bash` or `/usr/local/bin/bash`? I don't know...)